packages/xcm-provider/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@oak-network/xcm-provider",
```

Comment:
This provider is chain-provider but not xcm-provider. Our sdk is made for automation, which includes both native and xcm automations.